Mlem/App/Views/Shared/ExpandedPostView.swift

```diff
@@ -14,37 +14,34 @@ struct ExpandedPostView: View {
     @Environment(\.dismiss) var dismiss
     
     let post: AnyPost
+    @State var showCommentWithId: Int?
     @State var comments: [CommentWrapper] = []
     @State var loadingState: LoadingState = .idle
     
     var body: some View {
         ContentLoader(model: post) { proxy in
             if let post = proxy.entity {
-                content(for: post)
-                    .task {
-                        guard loadingState == .idle else { return }
-                        loadingState = .loading
-                        do {
-                            let comments = try await post.getComments(sort: .top, page: 1, maxDepth: 8, limit: 50)
-                            
-                            var output: [CommentWrapper] = []
-                            var keyedById: [Int: CommentWrapper] = [:]
-                            
-                            for comment in comments {
-                                let wrapper: CommentWrapper = .init(comment)
-                                keyedById[comment.id] = wrapper
-                                if let parentId = comment.parentCommentIds.last {
-                                    keyedById[parentId]?.addChild(wrapper)
-                                } else {
-                                    output.append(wrapper)
-                                }
-                            }
-                            self.comments = output
-                            loadingState = .done
-                        } catch {
-                            handleError(error)
-                        }
+                let showLoadingSymbol = showCommentWithId == nil || self.post.isUpgraded && loadingState != .loading
```

Comment:
*nit*

Some parentheses are nice for clarity here, since `&&` has precedence over `||` but appears later

```suggestion
                let showLoadingSymbol = showCommentWithId == nil || (self.post.isUpgraded && loadingState != .loading)
```